docs/userGuide/tweakingThePageStructure.md

```diff
@@ -250,7 +250,7 @@ A layout consists of the following files:
 1. Tweaks to the `<head>` (`head.md`)
 1. A site navigation menu (`navigation.md`)
 1. Custom styles (`styles.css`)
-1. Custom scripts (`scripts.js`)
+1. [Custom scripts (`scripts.js`)](#custom-javascript-in-scripts.js)
```

Comment:
Not necessary to use a hyperlink here.
1. For style consistency with the other list items.
2. On the latest master, page navigation will capture the level 3 heading for Custom Scripts.